utils/lib/grid_mapping.sql

```diff
@@ -0,0 +1,12 @@
+-- Grid location map
+-- This table maps locations from the input (physical) grid to the VPR grid
+-- locations and vice versa. If there are multiple entries for the same physical
+-- location (grid_phy_x, grid_phy_y) then it means that a single physical tile 
```

Comment:
Why not make this table using primary keys?  Denormalization exists for performance, I'm not sure it is required here.

Reply:
You mean not to store coordinates but pkeys (as foreign keys) of `phy_tile` and `tile` ?

Reply:
Yes.

Reply:
Every tile and phy_tile should appear in this map at least once.  That doesn't change my conclusion about the coordinates vs foreign keys.

Reply:
So in the end there is a table with pkey pairs.